tests/mpi/periodicity_01.with_petsc=true.mpirun=3.output.2

```diff
@@ -0,0 +1,163 @@
+
+Testing for dim=2
+
+Cycle 0:
+DEAL::Starting value 0.00436405
```

Comment:
@bangerth these outputs are different in 'Starting value` and `Convergence step` as different preconditioner is used. I could modify the code not to output this info and then one wouldn't need a separate output file. Do you think it's better that way?